base/dcp_client.go

```diff
@@ -437,7 +437,7 @@ func (dc *DCPClient) openStream(vbID uint16, maxRetries uint32) (err error) {
 			InfofCtx(logCtx, KeyDCP, "Open stream for vbID %d failed due to rollback or range error, will roll back metadata and retry: %v", vbID, openStreamErr)
 			err := dc.rollback(logCtx, vbID)
 			if err != nil {
-				WarnfCtx(logCtx, "failed to rollback metadata for vb %d: error: %v", vbID, err)
+				PanicfCtx(logCtx, "failed to rollback metadata for vb %d: error: %v", vbID, err)
```

Comment:
Is it possible to have the decision to panic left to a particular DCP stream type?  Or are there no feeds that it's better to roll back to zero?